docs/quickstart/install-and-use-a-package-in-visual-studio-mac.md

```diff
@@ -0,0 +1,102 @@
+
+For this walkthrough, use a simple .NET Core Console app. Create a project in Visual Studio for Mac using **File > New Solution...**, select the **.NET Core > App > Console Application** template. Click **Next**. Accept the default values for **Target Framework** when prompted.
+
+Visual Studio creates the project, which opens in Solution Explorer.
+
+## Add the Newtonsoft.Json NuGet package
+
+To install the package, you use the NuGet Package Manager. When you install a package, NuGet records the dependency in  either your project file or a `packages.config` file (depending on the project format). For more information, see [Package consumption overview and workflow](../consume-packages/Overview-and-Workflow.md).
+
+### NuGet Package Manager
+
+1. In Solution Explorer, right-click **Dependencies** and choose **Add Packages...**.
+
+    ![Manage NuGet Packages command for project References](media/QS_Use_Mac-02-ManageNuGetPackages.png)
+
+1. Choose "nuget.org" as the **Package source** in the top left corner of the dialog, and search for **Newtonsoft.Json**, select that package in the list, and select **Add Packages...**:
+
+    ![Locating Newtonsoft.Json package](media/QS_Use_Mac-03-NewtonsoftJson.png)
+
+    If you want more information on the NuGet Package Manager, see [Install and manage packages using Visual Studio for Mac](../consume-packages/install-use-packages-visual-studio.md).
+
+## Use the Newtonsoft.Json API in the app
+
+With the Newtonsoft.Json package in the project, you can call its `JsonConvert.SerializeObject` method to convert an object to a human-readable string.
+
+1. Open the `Program.cs` file (located in the Solution Pad) and replace the file contents with the following code:
+
+    ```cs
+    using System;
+    using Newtonsoft.Json;
+
+    namespace NuGetDemo
+    {
+        public class Account
+        {
+            public string Name { get; set; }
+            public string Email { get; set; }
+            public DateTime DOB { get; set; }
+        }
+    
+        class Program
+        {
+            static void Main(string[] args)
+            {
+                Account account = new Account()
+                {
+                    Name = "Joe Doe",
+                    Email = "joe@test.com",
+                    DOB = new DateTime(1976, 3, 24)
+                };
+                string json = JsonConvert.SerializeObject(account);
+                Console.WriteLine(json);
+            }
+        }
+    }
+    ```
+
+1. Build and run the app by selecting **Run > Start Debugging**:
+
+1. Once the app runs, you'll see the serialized JSON output appear in the console:
+
+  ![Output of the Console app](media/QS_Use_Mac-06-AppStart.png)
+
+## Next steps
+Congratulations on installing and using your first NuGet package!
+
+> [!div class="nextstepaction"]
+> [Install and manage packages using Visual Studio for Mac](/visualstudio/mac/nuget-walkthrough?toc=/nuget/toc.json)
```

Comment:
@jmatthiesen - what is the purpose of "?toc=/nuget/toc.json" in that URL?

Reply:
@Mikejo5000 provided that URL, my guess is that this is what does the trick where the link to the VSMac doc shows up with the NuGet TOC when followed. Mike - is that correct?

Reply:
Yes. Since they're just getting started with NuGet, presumably, it seems good to keep them in the NuGet doc view at this point. But it's also just fine to remove the toc.json reference and let them go to VS for Mac. Either way.

Reply:
sounds good! Thanks!